keras/backend.py

```diff
@@ -3389,7 +3389,8 @@ def resize_images(x, height_factor, width_factor, data_format,
       height_factor: Positive integer.
       width_factor: Positive integer.
       data_format: One of `"channels_first"`, `"channels_last"`.
-      interpolation: A string, one of `nearest` or `bilinear`.
+      interpolation: A string, one of `area`, `bicubic`, `bilinear`,
```

Comment:
To match the description of `data_format`, use:

```
A string, one of `"nearest"`, `"area"`, `"bicubic"`, `"bilinear"`,
`"gaussian"`, `"lanczos3"`, `"lanczos5"`, `"mitchellcubic"`.
Defaults to `"nearest"`.
```

Reply:
Changed in [Fix string formatting](https://github.com/keras-team/keras/pull/16249/commits/aa8af10a160845df30a8b28470e866e02c68bdf5)